src/Popover/Popover.js

```diff
@@ -380,7 +380,7 @@ const Popover = React.createClass({
 
   render() {
     return (
-      <div>
+      <div style={{display: 'none'}}>
```

Comment:
I wondered, but seemed like overkill given there are no other styles in that component.